packages/wrangler/src/__tests__/r2/bucket.test.ts

```diff
@@ -15,8 +15,6 @@ const mockGetR2BucketMetrics = vi.mocked(getR2BucketMetrics);
 
 describe("r2 bucket info", () => {
 	beforeEach(() => {
-		vi.resetAllMocks();
```

Comment:
This was undoing the console mocking above and is unnecessary.
Our vitest runs are configured to automatically reset mocks between tests. We should catch these in PR reviews in the future.

If mocks need to be reset within a test then those should be reset explicitly, and not rely upon this catch-all.